.github/workflows/periodic.yaml

```diff
@@ -0,0 +1,50 @@
+# Workflow to create an auto-buildable release periodically.
+#
+# Releases created with automation credentials don't trigger workflows.
+# Therefore a token `secrets.RELEASE_TOKEN` must exist with public_repo scope.
+name: Periodic release
+on:
+  # Run weekly on sunday at 21:37 UTC (arbitrary)
+  schedule:
+  - cron: '37 21 * * SUN'
+  # Allow triggering manually:
+  # curl -XPOST -d '{"event_type":"periodic"}' \
+  #  "-HAuthorization: Bearer <token>" \
+  #   https://api.github.com/repos/clangd-build/empty/dispatches
+  repository_dispatch: { types: periodic }
```

Comment:
the [documentation](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#example) says: 
`This event does not have additional activity types so it does not support the types keyword.`

are we sure this is legit?

Reply:
Docs are out of date, I've tested and it works (iff you send the matching event_type).

Here's the doc for the request: https://developer.github.com/v3/repos/#create-a-repository-dispatch-event
> event_type | string | Required: A custom webhook event name.